solr/solr-ref-guide/src/configuring-solr-xml.adoc

```diff
@@ -543,7 +553,6 @@ If only `dividend` routing is desired, `hash` may be explicitly set to the empty
 The `<metrics>` element in `solr.xml` allows you to customize the metrics reported by Solr.
 You can define system properties that should not be returned, or define custom suppliers and reporters.
 
-In a default `solr.xml` you will not see any `<metrics>` configuration.
```

Comment:
I removed this line from docs, as obviously there is now a `<metrics>` tag in our default solr.xml :)